packages/EasyTest/tests/CheckCoverageCommandTest.php

```diff
@@ -40,26 +64,33 @@ public function providerCheckCoverage(): iterable
      */
     public function providerCheckCoverageExceptions(): iterable
     {
-        yield 'File not found' => [
+        yield 'Txt file not found' => [
             [
                 'file' => 'invalid-file.txt',
             ],
             UnableToLoadCoverageException::class,
         ];
 
-        yield 'File but no coverage' => [
+        yield 'Txt file but no coverage' => [
             [
                 'file' => __DIR__ . '/fixtures/no-coverage.txt',
             ],
             UnableToResolveCoverageException::class,
         ];
 
-        yield 'File and Lines but no coverage' => [
+        yield 'Txt file and Lines but no coverage' => [
             [
                 'file' => __DIR__ . '/fixtures/lines-but-no-coverage.txt',
             ],
             UnableToResolveCoverageException::class,
         ];
+
+        yield 'Clover file but wrong format' => [
```

Comment:
What about a "not found" case for Clover? Or about to not split the "not found" case for Clover and Txt?

Reply:
It does not matter for [resolver](https://github.com/eonx-com/easy-monorepo/blob/master/packages/EasyTest/src/Coverage/Loaders/FilesystemCoverageLoader.php#L18) which extension nonexistent file has, so I decided not to add such case.

Reply:
Yep, I agree, And that's why I'm talking about to not split these cases. I.e. we can name the first key as `File not found` instead of `Txt file not found`. It would be more clear I believe.

Reply:
Ok, I've renamed it.